### PR TITLE
More types

### PR DESCRIPTION
**Pull request by mamazu**

### CODE REVIEW

File: lib/Extension/ClassMover/Application/ClassReferences.php [hunk 1]
```diff
@@ -13,6 +13,15 @@
 use Phpactor\Filesystem\Domain\FilesystemRegistry;
 use Phpactor\TextDocument\TextDocumentBuilder;
 
+/** @phpstan-type ReferenceArray array{
```
Comment (dantleech) [inline lib/Extension/ClassMover/Application/ClassReferences.php:16]: would be good to add support for this to Phpactor.

**Review by dantleech — APPROVED** [review]

File: lib/Extension/ClassMover/Application/ClassReferences.php (hunk 1 above)
Comment (mamazu) [reply lib/Extension/ClassMover/Application/ClassReferences.php:16]: Should I create an issue for this?

**Review by dantleech — APPROVED** [review]

**Comment by dantleech** [comment]
:tada:

File: lib/Extension/ClassMover/Application/ClassReferences.php (hunk 1 above)
Comment (dantleech) [reply lib/Extension/ClassMover/Application/ClassReferences.php:16]: :shrug:

**Pull request merged**